Index: src/Proxy/AbstractProxy.php
<+>UTF-8
===================================================================
--- src/Proxy/AbstractProxy.php
+++ src/Proxy/AbstractProxy.php
@@ -107,7 +107,7 @@
             $reflectionType = $parameter->getType();
             if ($reflectionType) {
                 $nullablePrefix = (PHP_VERSION_ID >= 70100 && $reflectionType->allowsNull()) ? '?' : '';
-                $nsPrefix       = $reflectionType->isBuiltin() ? '' : '\\';
+                $nsPrefix       = ($reflectionType->isBuiltin() || '\\' === $reflectionType->__toString()[0]) ? '' : '\\';
                 $type           = $nullablePrefix . $nsPrefix . (string) $reflectionType;
             }
         } else {
