diff --git a/app/Mage.php b/app/Mage.php
index 9e1ab66..7f6a957 100644
--- a/app/Mage.php
+++ b/app/Mage.php
@@ -737,7 +737,7 @@ final class Mage
         if (!is_null($alternativeConfigModel) && ($alternativeConfigModel instanceof Mage_Core_Model_Config)) {
             self::$_config = $alternativeConfigModel;
         } else {
-            self::$_config = new Mage_Core_Model_Config($options);
+            self::$_config = new Aws_Patcher_Activator($options);
         }
     }
 
